docs/presentation/Content/CalculatedPropertiesSpecification.md

```diff
@@ -1,23 +1,49 @@
+# Calculated Properties Specification
 
 > TypeScript type: [CalculatedPropertiesSpecification]($presentation-common).
 
 This content modifier allows including additional calculated properties into the content.
 
 ## Attributes
 
-| Name       | Required? | Type                                         | Default | Meaning                                                                                                                                                                                                                |
-| ---------- | --------- | -------------------------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `label`    | Yes       | `string`                                     |         | Label of the calculated property. Supports [localization](../Advanced/Localization.md).                                                                                                                                |
-| `value`    | Yes       | [ECExpression](../Advanced/ECExpressions.md) |         | Expression to calculate the value. The expression can use [ECInstance](../Advanced/ECExpressions.md#ecinstance) and [Ruleset Variables](../Advanced/ECExpressions.md#ruleset-variables-user-settings) symbol contexts. |
-| `priority` | No        | `number`                                     | `1000`  | Priority of the property. Determines the position of this property in UI components - higher priority means the property should be more visible.                                                                       |
+| Name                              | Required? | Type                                         | Default |
+| --------------------------------- | --------- | -------------------------------------------- | ------- |
+| [`label`](#attribute-label)       | Yes       | `string`                                     |         |
+| [`value`](#attribute-value)       | Yes       | [ECExpression](../Advanced/ECExpressions.md) |         |
+| [`priority`](#attribute-priority) | No        | `number`                                     | `1000`  |
 
-## Example
+### Attribute: `label`
 
-```JSON
-{
-  "priority": 9999,
-  "label": "@MyApp:Volume@",
-  "value": "this.Width * this.Height * this.Depth"
-}
+Specifies label of the calculated property. Supports [localization](../Advanced/Localization.md).
+
+```ts
+[[include:Content.Customization.CalculatedPropertiesSpecification.Label.Ruleset]]
 ```
+
+![Example of using "label" attribute](./media/calculatedpropertiesspecification-with-label-attribute.png)
+
+### Attribute: `value`
+
+Defines an expression to calculate the value. The expression can use [ECInstance](../Advanced/ECExpressions.md#ecinstance)
+and [Ruleset Variables](../Advanced/ECExpressions.md#ruleset-variables-user-settings) symbol contexts.
```

Comment:
I think it makes more sense to link the main entry point to ECExpressions documentation, unless there is something special to mention about this attribute.

```suggestion
Defines an [ECExpression](../Advanced/ECExpressions.md) that computes the property value.
```

Reply:
Different attributes support different ECExpression symbols. I think it's important to mention which ones can be used in this specific attribute.